ports/calceph/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "calceph",
+  "version-semver": "3.5.0",
```

I'm sorry to give you a conflicting ask: did upstream say that they actually do semver somewhere or should this just be `version`?

```suggestion
  "version": "3.5.0",
```

---

This line was modified by the other reviewer in this commit : 
https://github.com/microsoft/vcpkg/commit/747af1892de6671e6fb2d8b294c9d70b439b7470
What is the applied rule for the naming in the documentation?

---

I apologize for my previous confusion about your review. semver is a statement about how the port will change its version number over time in response to changes in its products. My previous suggestions for your review were not accurate enough.There is the offical [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions)

---

Thanks for the clarification